client/src/LayoutComponents/ProjectDetails/ProjectDetails.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import './ProjectDetails.css';
+import github from '../../images/github.svg';
+import pc from '../../images/pc.svg';
+import ProjectView from '../../SharedComponents/ProjectView';
+import StudentCard from '../../SharedComponents/StudentCard';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+class ProjectDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      projects: [],
+    };
+  }
+  componentDidMount() {
+    const { id } = this.props;
+    // id coming from cohorts and studentProfile pages
+    axios
+      .get(`api/project/${id}`)
+      .then((res) => {
+        const resultArray = res.data.map((e) => e);
```

Comment:
@kholoudfann I don't think the map() is actually doing anything, so you could just have:
`const resultArray = res.data;`